jest.config.ts

```diff
@@ -11,6 +11,17 @@ export default () => {
   return {
     clearMocks: true,
     preset: 'ts-jest',
+    globals: {
+      'ts-jest': {
+        tsconfig: 'tsconfig.jest.json',
+        diagnostics: false,
+      },
+    },
+    moduleNameMapper: {
+      '.+\\.(css|styl|less|sass|scss)$': 'identity-obj-proxy',
+      '.+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
```

Comment:
@michalsidzej please look at update jest config as you fixed before issues connected with the images in the tests